zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/client/DocumentenApiClient.kt

```diff
@@ -149,6 +151,95 @@ class DocumentenApiClient(
             }
     }
 
+    fun lockInformatieObject(
+        authentication: DocumentenApiAuthentication,
+        objectUrl: URI
+    ): DocumentLock {
+        val result = checkNotNull(
+            webclientBuilder
+                .clone()
+                .filter(authentication)
+                .filter(ClientTools.zgwErrorHandler())
+                .build()
```

Comment:
Up until the `.build()`, there's a lot of duplicated code in this class. Can we separate it to a private method?

Reply:
Fixed